10:19:59.269 [main] DEBUG com.cohort4.cohort4jaxrsapp.dao.UserDao - This is a DEBUG log
10:19:59.276 [main] TRACE com.cohort4.cohort4jaxrsapp.dao.UserDao - This is a TRACE log
10:19:59.693 [main] INFO  com.cohort4.cohort4jaxrsapp.config.DbConnection - DB SUCCESS :  Connection Successful
10:20:30.296 [main] ERROR com.cohort4.cohort4jaxrsapp.config.DbConnection - 52514534 DB ERROR :  Connection failed - Access denied for user 'root'@'localhost' (using password: YES)
10:53:09.899 [main] INFO  com.cohort4.cohort4jaxrsapp.config.DbConnection - DB SUCCESS :  Connection Successful
11:25:12.313 [main] INFO  com.cohort4.cohort4jaxrsapp.config.DbConnection - DB SUCCESS :  Connection Successful
11:25:30.380 [main] INFO  com.cohort4.cohort4jaxrsapp.config.DbConnection - DB SUCCESS :  Connection Successful
11:25:39.381 [main] INFO  com.cohort4.cohort4jaxrsapp.config.DbConnection - DB SUCCESS :  Connection Successful
11:53:34.618 [main] INFO  com.cohort4.cohort4jaxrsapp.config.DbConnection - DB SUCCESS :  Connection Successful
11:53:34.681 [main] INFO  com.cohort4.cohort4jaxrsapp.config.DbConnection - DB SUCCESS :  Connection Successful
